### ISSUE TITLE
No such file or directory db.sqlite-journal on Windows

### ISSUE THREAD

**Issue by linbudu599** [blitz-js/blitz#1652]
### What is the problem?

mutate operation will cause process Errors:

```bash
node:internal/process/promises:218
          ^

[Error: ENOENT: no such file or directory, stat 'D:\Blitz-Pratices\db\db.sqlite-journal'] {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'stat',
  path: 'D:\\Blitz-Pratices\\db\\db.sqlite-journal'
}
npm ERR! code 1
npm ERR! path D:\Blitz-Pratices
npm ERR! command failed
npm ERR! command C:\Windows\system32\cmd.exe /d /s /c "blitz start"
```

### Steps to Reproduce

1.  following the tutorial in the docs.
2.  any mutate operations will cause this error like `update` / `create` / `delete`.

### Versions

```
debug: blitzPkgPath: D:\Blitz-Pratices\node_modules\blitz\dist\index.js
debug: cliPkgPath: D:\Blitz-Pratices\node_modules\@blitzjs\cli\lib\src\index.js 

Windows 10 | win32-x64 | Node: v15.1.0

blitz: 0.27.1 (global)
blitz: 0.27.1 (local)

  Package manager: yarn
  System:
    OS: Windows 10 10.0.18363
    CPU: (16) x64 AMD Ryzen 7 4800H with Radeon Graphics
    Memory: 7.84 GB / 15.37 GB
  Binaries:
    Node: 15.1.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.10 - ~\AppData\Roaming\npm\yarn.CMD
    npm: 7.0.8 - C:\Program Files\nodejs\npm.CMD
    Watchman: Not Found
  npmPackages:
    @prisma/cli: ~2.12 => 2.12.1
    @prisma/client: ~2.12 => 2.12.1
    blitz: 0.27.1 => 0.27.1
    react: 0.0.0-experimental-4ead6b530 => 0.0.0-experimental-4ead6b530
    react-dom: 0.0.0-experimental-4ead6b530 => 0.0.0-experimental-4ead6b530
    typescript: 4.x => 4.1.3
```

### Other

Please include applicable logs and screenshots that show your problem.

logs in `debug.log` above

```log
0 verbose cli [
0 verbose cli   'C:\\Program Files\\nodejs\\node.exe',
0 verbose cli   'C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js',
0 verbose cli   'start'
0 verbose cli ]
1 info using npm@7.0.8
2 info using node@v15.1.0
3 timing config:load:defaults Completed in 2ms
4 timing config:load:file:C:\Program Files\nodejs\node_modules\npm\npmrc Completed in 2ms
5 timing config:load:builtin Completed in 2ms
6 timing config:load:cli Completed in 2ms
7 timing config:load:env Completed in 0ms
8 timing config:load:file:D:\Blitz-Pratices\.npmrc Completed in 1ms
9 timing config:load:project Completed in 1ms
10 timing config:load:file:C:\Users\96457\.npmrc Completed in 1ms
11 timing config:load:user Completed in 1ms
12 timing config:load:file:C:\Users\96457\AppData\Roaming\npm\etc\npmrc Completed in 4ms
13 timing config:load:global Completed in 4ms
14 timing config:load:cafile Completed in 0ms
15 timing config:load:validate Completed in 1ms
16 timing config:load:setUserAgent Completed in 1ms
17 timing config:load:setEnvs Completed in 1ms
18 timing config:load Completed in 17ms
19 verbose npm-session 895e55957393d5dc
20 timing npm:load Completed in 36ms
21 timing command:run-script Completed in 172801ms
22 timing command:start Completed in 172812ms
23 verbose stack Error: command failed
23 verbose stack     at ChildProcess.<anonymous> (C:\Program Files\nodejs\node_modules\npm\node_modules\@npmcli\promise-spawn\index.js:64:27)
23 verbose stack     at ChildProcess.emit (node:events:327:20)
23 verbose stack     at maybeClose (node:internal/child_process:1055:16)
23 verbose stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:288:5)
24 verbose pkgid demo@1.0.0
25 verbose cwd D:\Blitz-Pratices
26 verbose Windows_NT 10.0.18363
27 verbose argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "start"
28 verbose node v15.1.0
29 verbose npm  v7.0.8
30 error code 1
31 error path D:\Blitz-Pratices
32 error command failed
33 error command C:\Windows\system32\cmd.exe /d /s /c "blitz start"
34 verbose exit 1

```

**Comment by linbudu599** [comment]
and, though error occured, the mutate operation is performed successfully.

**Comment by linbudu599** [comment]
**supplement**: only the first time of the [Type] + [Mutation] will cause this error, for example: `Create Question` will only invoke the error once, and then no erros will happened again in `Create Question`. And `Vote Choice` `Update Question` ...all other mutations will still got errors on the first trigger.

**Comment by flybayer** [comment]
Hey @linbudu599, not sure what to say on this. Looks like `db.sqlite-journal` is a temporary file that sqlite may create (and that you can delete). 

Does the error always happen? What about if you stop the server and run `blitz start` again?

**Comment by linbudu599** [comment]
> Hey @linbudu599, not sure what to say on this. Looks like `db.sqlite-journal` is a temporary file that sqlite may create (and that you can delete).
> 
> Does the error always happen? What about if you stop the server and run `blitz start` again?

nope, I've tried many times to re-appear this error and locate the reason, but it will not always happen.
If this error occured, restart the terminal and rerun `blitz start` will not help, in the restarted application this error will always happen. But if I restart the IDE to restart the application, once it works normally, then this error will not occured again.

Supplement: this error never appears in Linux, so I think maybe it was caused by the **annoying windows permissions system**: SQLite is not able to create journal file to implement commit.
